libtf.h

```diff
@@ -62,6 +62,11 @@ int libtf_invoke(const unsigned char *model_data, // TensorFlow Lite binary mode
                  libtf_output_data_callback_t output_callback, // Callback to use the model output data byte array.
                  void *output_callback_data); // User data structure passed to output callback.
 
+int libtf_initialize_micro_features();
+
+int libtf_generate_micro_features(const int16_t* input, int input_size,
```

Comment:
Add comments about what the input and outputs are like all the other libtf methods.

Reply:
E.g. same type of comment block on it as lib_tf_invoke.